### PR TITLE
changes made in the App

### PR DESCRIPTION
**Pull request by utkarsh006**
- Updated the gradle files to the latest version
- Added descriptions of the functions
- Enhanced the README file
- The app is running successfully by doing these changes.

### CODE REVIEW

**Comment by utkarsh006** [comment]
@ezatpanah Kindly Review

**Comment by ezatpanah** [comment]
> @ezatpanah Kindly Review

Hi , Thanks for your contribute 😊 let me check the changes and review them and then will merge it ✨

File: app/src/main/java/com/ezatpanah/roomdatabase_youtube/db/NoteDatabase.kt
Comment (ezatpanah) [inline app/src/main/java/com/ezatpanah/roomdatabase_youtube/db/NoteDatabase.kt:1]: Please remove the link, If you want to refer something to some website, I recommend to use Android Developer website , that's is the standard resource

File: README.md
Comment (ezatpanah) [inline README.md:1]: Thanks for editing , Cool 🤩🙏

File: app/src/main/res/values/themes.xml
Comment (ezatpanah) [inline app/src/main/res/values/themes.xml:1]: Would you please back this one on default  [with action bar] , Thanks

**Review by ezatpanah — COMMENTED** [review]
Thanks for your effort and your collaborate on this project ✨🙏

File: README.md
Comment (utkarsh006) [reply README.md:1]: No issues :) My pleasure

File: app/src/main/java/com/ezatpanah/roomdatabase_youtube/db/NoteDatabase.kt
Comment (utkarsh006) [reply app/src/main/java/com/ezatpanah/roomdatabase_youtube/db/NoteDatabase.kt:1]: Yes changes done !

File: app/src/main/res/values/themes.xml
Comment (utkarsh006) [reply app/src/main/res/values/themes.xml:1]: Done

**Comment by utkarsh006** [comment]
> Thanks for your effort and your collaborate on this project ✨🙏

No issues, I am feeling good that I got a chance to help you :)

**Review by ezatpanah — COMMENTED** [review]
all looks good , again thank you

**Comment by utkarsh006** [comment]
> all looks good , again thank you

@ezatpanah Haha no need of Thanks 😄 Let's connect on LinkedIn

**Pull request merged**